Framework/Algorithms/test/ExportTimeSeriesLogTest.h

```diff
@@ -258,13 +258,13 @@ class ExportTimeSeriesLogTest : public CxxTest::TestSuite {
     Kernel::TimeSeriesProperty<double> *sinlog = new Kernel::TimeSeriesProperty<double>("FastSineLog");
     double period = static_cast<double>(pulsetime_ns);
     curtime_ns = runstarttime_ns;
-    size_t numevents = 0;
+    // size_t numevents = 0;
```

Comment:
Due to a clang update, counters which increment but aren't otherwise use give warnings. I've commented the counters out in this PR, as they could be useful for debugging. Alternatively, could have printed them out or removed them completely... open to suggestions.